client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts

```diff
@@ -363,6 +363,12 @@ export const meetingSettings: SettingsGroup[] = fillInSettingsDefaults([
                         label: _(`Enable interposed questions`),
                         type: `boolean`
                     },
+                    {
+                        key: `list_of_speakers_can_create_point_of_order_for_others`,
+                        label: _(`Enable point of orders for other participants`),
+                        helpText: _(`Requires permission can manage list of speaker`),
```

Comment:
either use the technical term `list_of_speakers.can_manage` (probably not, though) or form a full sentence like "Requires permission to manage lists of speakers"